packages/target-electron/src/index.ts

```diff
@@ -32,6 +32,7 @@ if (rc['help'] === true || rc['h'] === true) {
   process.exit()
 }
 
+import { callsWebappElectronScheme } from './windows/video-call.js'
 protocol.registerSchemesAsPrivileged([
```

Comment:
```suggestion
// Note: make sure to call registerSchemesAsPrivileged only once with all schemes
// you want to register, since following calls will override previous registered schemes
protocol.registerSchemesAsPrivileged([
```